core/player/BUILD

```diff
@@ -2,16 +2,22 @@ load("@rules_player//javascript:defs.bzl", "js_pipeline")
 load("@npm//:defs.bzl", "npm_link_all_packages")
 load("//tools:defs.bzl", "NATIVE_BUILD_DEPS", "tsup_config", "vitest_config")
 
+# TODO: Would be nice to macro all the native things to avoid missing this setup (consequence is bundling Player in a plugin bundle)
```

Comment:
Do you have any estimate for how much effort this would require? If its not huge would it be worth it to just do it now?